src/Readers/AbstractReader.js

```diff
@@ -40,12 +40,13 @@ class AbstractReader {
    * @return {{__FILE__, __PATH__}}
    */
   parsePath(source) {
-
     const __FILE__ = path.basename(source);
     let __PATH__;
 
     let parsedURL = url.parse(source);
-    if (parsedURL.protocol) {
+
+    // url parse can find protocol in Windows-style path, so we check it
+    if (parsedURL.protocol && !parsedURL.protocol.match(/^(\w\:)?$/i)) {
```

Comment:
It is not correct, cose in spec we have test for normalizing paths, in this case we didn't normalize our path and didn't pass the test.  I can add more complicated check for file path, to support single char protocol .